packages/why3find/why3find.1.0/opam

```diff
@@ -0,0 +1,51 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A Why3 Package Manager"
+description:
+  "The why3find utility is designed for managing packages for why3 developpers and associated OCaml extracted code."
+maintainer: ["benjamin.jorge@cea.fr" "loic.correnson@cea.fr"]
+authors: [
+  "Loïc Correnson <loic.correnson@cea.fr>"
+  "Benjamin Jorge <benjamin.jorge@cea.fr>"
+]
+license: "LGPL-2.1"
```

Comment:
There's a lint warning here. The standard for licenses allows the following

```suggestion
license: "LGPL-2.1-only"
```

or

```suggestion
license: "LGPL-2.1-or-later"
```

Which is this released under?

Reply:
LGPL-2.1-only